johnmusiu/Maintenance-Tracker
# #158052672 API refactor folder structure

### What does this PR do?
Refactors folder structure, adds blueprints

### Description of Task to be completed?
Changes the app structure by moving routes code from `__init__.py` into the relevant blueprint folders
Moves `models.py` from `api/app/` into `api/`
Adds misplaced Procfile

### How should this be manually tested?
Browse through [this repository](https://github.com/johnmusiu/Maintenance-Tracker/tree/ch-api-refactor-folder-structure-158052672) to see the new structure

### What are the relevant pivotal tracker stories?
[#158052672](https://www.pivotaltracker.com/story/show/158052672)
